types/parse/index.d.ts

```diff
@@ -1,6 +1,8 @@
-// Type definitions for parse 1.9
+// Type definitions for parse 2.2.20
```

Comment:
As far as I get it only the first two numbers (2.2) matter; the last number will be used for fixes to the typings.